CHANGELOG.md

```diff
@@ -43,6 +43,7 @@
 * [FEATURE] PromQL: Add experimental `info` function. Experimental functions are disabled by default, but can be enabled setting `-querier.promql-experimental-functions-enabled=true` in the query-frontend and querier. #9879
 * [FEATURE] Distributor: Support promotion of OTel resource attributes to labels. #8271
 * [FEATURE] Querier: Add experimental `double_exponential_smoothing` PromQL function. Experimental functions are disabled by default, but can be enabled by setting `-querier.promql-experimental-functions-enabled=true` in the query-frontend and querier. #9844
+* [FEATURE] Distributor: Add experimental `memberlist` kvStore for ha_tracker. Memberlist parameters can be changed through `-memberlist-*` flags
```

Comment:
can you also add this in the `about-versioning.md` doc? It's where we list all experimental features. You can also explain briefly why it is considered experimental.

also for this entry: can you add the PR number and also mention the full config option by which they can use memberlist for the HA tracker.

Reply:
Thanks a lot @dimitarvdimitrov  for the suggestion. I have applied it [55d6ab3](https://github.com/grafana/mimir/pull/10054/commits/55d6ab371eb5c123f57b5a8f06cfad2b82065034).